Lib/importlib/_bootstrap.py

```diff
@@ -1156,8 +1154,7 @@ def _find_and_load(name, import_):
         _lock_unlock_module(name)
 
     if module is None:
-        message = ('import of {} halted; '
-                   'None in sys.modules'.format(name))
+        message = (f'import of {name} halted; None in sys.modules')
```

Comment:
The parens here can be deleted; it looks like they were previously used for the multi-line string.